src/Prelude.hs

```diff
@@ -1,7 +1,48 @@
--- | Uses [relude](https://github.com/kowainik/relude) as default Prelude.
+{-# OPTIONS -fno-warn-orphans #-}
```

Comment:
This option can be removed from this file, we don't have orphan instances here for now.

Reply:
Got it.